deployment/network-operator/README.md

```diff
@@ -285,7 +285,7 @@ You have to wait for network-operator POD to remove before proceeding.
 ### Apply Helm chart update
 
 ```
-helm upgrade -n network-operator  network-operator mellanox/network-operator --version=<VERSION> -f values-<VERSION>.yaml
+helm upgrade -n network-operator  network-operator mellanox/network-operator --version=<VERSION> -f values-<VERSION>.yaml --force
```

Comment:
@rollandf since helm will use a replace when force is provided. it will use a replace strategy[1]
how will this affect MOFED container ? will the ds be deleted and recreated ?

if so, i still believe we will be OK, with caveat that we will go through inbox driver first.
(driver container deleted, new ds starts, mofed upgrade flow from inbox kicks in)

that said we need to merge this first.

[1] https://github.com/helm/helm/blob/82195652495d757d87c1a01f640973f20ca141b0/pkg/kube/client.go#L672

Reply:
Not sure I see why it should affect the MOFED container.
If the MOFED version change, the image will be updated, but the pods are not restarted (only onDelete).

Reply:
this is not an issue you are right.